tools/bin/goreleaser_utils

```diff
@@ -71,6 +72,15 @@ get_remote_plugin_paths() {
   done
 }
 
+build_standard_capabilities() {
+  cd ./capabilities
+  npx nx@19.8.2 init
+  ./nx run-many -t build
```

Comment:
@HenryNguyen5 @chainchad should I be using `actions/cache` to cache these binaries?

Reply:
Also no manifest right now. Is a manifest a requirement of phase 1? Or can it come with phase 2 as we harden?

Reply:
WDYM by manifest?

Reply:
The setup-go action should already cache the intermediate artifacts to the binaries, if not the binaries themselves. Probably best to leave it as-is. If build times continue to be an issue, it's better to publish the binaries on the capabilities repo side then pull down the pre-built binaries in here.

Reply:
> WDYM by manifest?

Originally @chainchad @cedric-cordenier and I talked about the first iteration of this having:

"
Some manifest file which has the plugin name, version, url, and checksum. For ex.
```
version: 1.0
dependencies:
- name: dependency1
version: 1.2.3
url: [https://example.com/dependency1/v1.2.3/dependency1.tar.gz](https://www.google.com/url?q=https://example.com/dependency1/v1.2.3/dependency1.tar.gz&sa=D&source=docs&ust=1734635936940744&usg=AOvVaw16wDn9XzQdSKoTtxHG6Po1)
checksum: sha256:abcdef1234567890abcdef1234567890abcdef1234567890abcdef1234567890
```
"

But currently it just builds from the latest head in the capabilities repo without doing any checks

Reply:
got it, makes sense

Reply:
If we don't have a manifest, the implication is that we'll always build this with the standard capabilities? I suppose that's fine for this image which is a "develop" image. If we want to release this, I suppose we'll need that manifest then. Would that work?

Reply:
> If we don't have a manifest, the implication is that we'll always build this with the latest standard capabilities? I suppose that's fine for this image which is a "develop" image. If we want to release this, I suppose we'll need that manifest then. Would that work?

Yes. That plan sounds good to me.